src/errors.ts

```diff
@@ -2,6 +2,11 @@ export const UNEXPECTED_ERROR_CODE = 'UNEXPECTED_ERROR';
 export const UNEXPECTED_ERROR_REASON =
   'Unexpected error occurred executing integration! Please contact us in Slack or at https://support.jupiterone.io if the problem continues to occur.';
 
+export const PROVIDER_AUTH_ERROR_DESCRIPTION =
+  ' Failed to access provider resource.' +
+  ' This integration is likely misconfigured or has insufficient permissions required to access the resource.' +
+  " Please ensure your integration's configuration settings are setup correctly.";
```

Comment:
This is super minor, but "setup" is technically a noun. I think "set up" is correct wording.